spec/lib/secure_headers_spec.rb

```diff
@@ -196,6 +198,38 @@ def set_security_headers(subject)
     end
   end
 
+  describe "#set_public_key_pins" do
+    it "sets the Public-Key-Pins header" do
+      should_assign_header(HPKP_HEADER_NAME + "-Report-Only", "max-age=1234")
```

Comment:
yes -- only `max_age` is required. from http://tools.ietf.org/html/draft-ietf-websec-key-pinning-21#section-2.1.1

"""According to the processing rules of Section 2.1, the UA MUST ignore
   pin-directives with tokens naming hash algorithms it does not
   recognize.  If the set of remaining effective pin-directives is
   empty, and if the host is a Known Pinned Host, the UA MUST cease to
   consider the host as a Known Pinned Host (the UA should fail open)."""